tests/out/wgsl/pointers.wgsl

```diff
@@ -11,9 +6,3 @@ fn f() {
     return;
 }
 
-fn index_dynamic_array(p: ptr<workgroup, DynamicArray>, i: i32, v_1: u32) -> u32 {
```

Comment:
can we just switch this to "storage" instead of removing?

Reply:
Well, that's the thing: you can't pass `ptr<storage, T>` to user-defined functions. There is no way to pass a pointer to any unsized type to a function.

What we can do, instead, is test using pointers to dynamically sized arrays, within a function. I can add a test for that.

Reply:
Right, yes, we should have this test still. Thank you!

Reply:
This change is done, but it depends on #1543.

Reply:
(See `tests/in/pointers.wgsl`, for the new pointer-to-unsized test.)